File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.js
/* /dev/stdin */
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().split('\n')

const [n, m] = input[0].split(' ').map(Number)



let result = '';

for(let i = 1; i <= n; i++) {
  const a = input[i].split(' ').map(Number);
  const b = input[i+n].split(' ').map(Number);

  for(let j = 0; j < a.length; j++) {
    result += (a[j]+b[j]) + ' '
  }
  result += '\n'
  
}

console.log(result)

File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.py
import sys

N, M = map(int, sys.stdin.readline().strip().split(' '))

arrIn = []
result = ''

for _ in range(N*2):
  arrIn.append(sys.stdin.readline().strip())


for i in range(N):
  a = list(map(int, arrIn[i].split(' ')))
  b = list(map(int, arrIn[i+N].split(' ')))

  for j in range(M):
    result += str(a[j]+b[j]) + ' '

  result += '\n'

print(result)
  